src/data/abilities/ab-attrs/synchronize-status-ab-attr.ts

```diff
@@ -10,37 +10,36 @@ import type { Pokemon } from "#field/pokemon";
  * Used for {@linkcode https://bulbapedia.bulbagarden.net/wiki/Synchronize_(Ability) | Synchronize}.
  */
 export class SynchronizeStatusAbAttr extends AbAttr {
-  constructor(showAbility: boolean = true, showAbilityInstant: boolean = false) {
-    super(showAbility, showAbilityInstant);
+  /** All status effects that can be transferred by Synchronize */
+  private static syncStatuses = new Set<StatusEffect>([
```

Comment:
Same comment as in `MoveEffectChanceMultiplierAbAttr` re: private static fields -> un-exported `const` outside the class.
```suggestion
  private static readonly syncStatuses = new ReadonlySet<StatusEffect>([
```